fstat/templates/index.html

```diff
@@ -40,17 +40,56 @@
     </div>
     <div class="table-responsive">
       <table class="table table-bordered">
-        <tr>
-          <th>Signature</th>
-          <th>Failure Count</th>
-        </tr>
-        {% for f, count in failures.most_common(total) %}
-        <tr>
-          <td><a href="http://git.gluster.org/cgit/glusterfs.git/tree/{{ f.signature }}" target="_blank"><code>{{ f.signature }}</code></a></td>
-          <td><a href="{{ url_for('instance_summary', start_date=start_date, end_date=end_date, fid=f.id, branch=request.args.get('branch', 'all')) }}">{{ count }}</a></td>
-        <tr>
-        {% endfor %}
-      </table>
-  </div>
+      <tr>
+        <th>Signature</th>
+        <th>Failure Count</th>
+        <th>Associated Bugs</th>
+      </tr>
+      {% for f, count in failures.most_common(total) %}
+      <tr>
+        <td><a href="http://git.gluster.org/cgit/glusterfs.git/tree/{{ f.signature }}" target="_blank"><code>{{ f.signature }}</code></a></td>
+        <td><a href="{{ url_for('instance_summary', start_date=start_date, end_date=end_date, fid=f.id, branch=request.args.get('branch', 'all')) }}">{{ count }}</a></td>
+        <td>
+          {% if not f.bugs %}
+            No Bugs Associated
```

Comment:
Don't show this line. Just showing the pen icon should be enough. We're likely to have empty fields more often than not.